crap-advisor-api/src/main/kotlin/com/aiecel/crapadvisor/api/controller/RestroomController.kt

```diff
@@ -1,34 +1,36 @@
 package com.aiecel.crapadvisor.api.controller
 
 import com.aiecel.crapadvisor.api.model.AddRestroomRequest
-import com.aiecel.crapadvisor.mapper.LocationMapper
 import com.aiecel.crapadvisor.mapper.RestroomMapper
 import com.aiecel.crapadvisor.service.RestroomService
 import io.swagger.v3.oas.annotations.Operation
 import io.swagger.v3.oas.annotations.tags.Tag
 import org.springframework.validation.annotation.Validated
-import org.springframework.web.bind.annotation.*
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
 
 @RestController
 @RequestMapping("/restrooms")
 @Tag(name = "Restrooms")
 class RestroomController(
     private val service: RestroomService,
-    private val restroomMapper: RestroomMapper,
-    private val locationMapper: LocationMapper
+    private val mapper: RestroomMapper
 ) {
 
     @GetMapping
     @Operation(summary = "Get all restrooms")
-    fun getAll() = service.getAll().map { restroomMapper.map(it) }
+    fun getAll() = service.getAll().map { mapper.map(it) }
 
     @PostMapping
     @Operation(summary = "Save a new restroom")
     fun save(@RequestBody @Validated request: AddRestroomRequest) =
-        restroomMapper.map(
+        mapper.map(
```

Comment:
Glad you did it. Good

Reply:
one small step for a file, one giant leap for a whole project